src/main/scala/cromwell/backend/provider/local/LocalBackend.scala

```diff
@@ -136,19 +137,19 @@ class LocalBackend(task: TaskDescriptor) extends BackendActor with StrictLogging
     *
     * @return Return hash for related task.
     */
-  override def computeHash: String = {
+  override def computeHash: Future[ExecutionHash] = {
     val orderedInputs = task.inputs.toSeq.sortBy(_._1)
     val orderedOutputs = task.outputs.sortWith((l, r) => l.name > r.name)
     val orderedRuntime = ListMap(task.runtimeAttributes.toSeq.sortBy(_._1):_*)
     val overallHash = Seq(
       task.commandTemplate,
-      orderedInputs map { case (k, v) => s"$k=${caching.computeWdlValueHash(v)}" } mkString "\n",
+      orderedInputs map { case (k, v) => s"$k=${computeWdlValueHash(v)}" } mkString "\n",
       // TODO: Docker hash computation is missing. In case it exists.
       orderedRuntime map { case (k, v) => s"$k=$v" } mkString "\n",
       orderedOutputs map { o => s"${o.wdlType.toWdlString} ${o.name} = ${o.requiredExpression.toWdlString}" } mkString "\n"
     ).mkString("\n---\n")
 
-    DigestUtils.md5Hex(overallHash)
+    Future.successful(ExecutionHash(DigestUtils.md5Hex(overallHash)))
```

Comment:
You are creating this future due to (TO BE ADDED) Docker hashing functionality? If not, I don't see why to use a future...